l10n_th_partner/views/res_users_view.xml

```diff
@@ -11,5 +11,13 @@
             </xpath>
         </field>
     </record>
-
+        <record id="view_partner_form_firstname" model="ir.ui.view">
```

Comment:
This is user file not partner file, you can move this record to res_partner_view.xml

Reply:
@newtratip  Thank you. I have just moved those lines.